### PR TITLE
Teleport 6.0 Release Notes

### PR DESCRIPTION
**Pull request by russjones**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,37 +1,40 @@
 # Changelog
 
-## 6.0.0-rc.1
+## 6.0.0
 
-**Note:** This version is a pre-release and is not recommended for production usage.
+Teleport 6.0 is a major release with new features, functionality, and bug fixes.
 
-This release of Teleport contains a number of improvements and bug fixes.
+The major features are [Database Access](https://goteleport.com/teleport/docs/preview/database-access/), open source role-based access control (RBAC), and exposing a easy to use API.
```
Comment (webvictim) [inline CHANGELOG.md:7]:
```suggestion
The major features are [Database Access](https://goteleport.com/teleport/docs/preview/database-access/), open source role-based access control (RBAC), and exposing an easy to use API.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,37 +1,40 @@
...
-* Enable proxy protocol support for Teleport Kubernetes proxy service: [#5299](https://github.com/gravitational/teleport/pull/5299).
-* Update `tsh db ls` command to include connection information and add `tsh db config` command: [#5319](https://github.com/gravitational/teleport/pull/5319).
-* Fix authentication failure when logging in via CLI with Access Workflows after removing `.tsh` directory: [#5323](https://github.com/gravitational/teleport/pull/5323).
-* Rename Database Access service `database` role to `db`: [#5359](https://github.com/gravitational/teleport/pull/5359).
-* Fix `tsh login` failure when `--proxy` differs from actual proxy public address: [#5380](https://github.com/gravitational/teleport/pull/5380).
+## Improvements
 
-## 6.0.0-alpha.1
+* Added `utmp`/`wtmp` support for SSH in [#5491](https://github.com/gravitational/teleport/pull/5491).
+* Added the ability to set a Kubernetes specific public address in [#5611](https://github.com/gravitational/teleport/pull/5611).
```
Comment (webvictim) [inline CHANGELOG.md:22]: You could actually do this before; this PR just makes the setting more obvious.

```suggestion
* Added the ability to set a Kubernetes specific public address in [#5611](https://github.com/gravitational/teleport/pull/5611).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,37 +1,40 @@
...
+* Added Proxy Protocol support to Kubernetes Access in [#5299](https://github.com/gravitational/teleport/pull/5299).
+* Added ACME ([Let's Encrypt](https://letsencrypt.org/)) support to make getting and using TLS certificates easier. [#5177](https://github.com/gravitational/teleport/issues/5177).
```
Comment (webvictim) [inline CHANGELOG.md:24]: We should add docs/details about how to use this functionality when setting up a new Teleport cluster. Do we have these written @klizhentas?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,37 +1,113 @@
...
+controls for databases, and capture SQL query activity in the audit log.
+
+#### Getting Started
+
+Configure Database Access from scratch following the [Getting Started](https://goteleport.com/teleport/docs/database-access/getting-started.mdx)
+guide.
+
+#### Guides
+
+* [AWS RDS/Aurora PostgreSQL](https://goteleport.com/teleport/docs/database-access/guides/postgres-aws.mdx)
```
Comment (r0mant) [inline CHANGELOG.md:28]: Note: none of these links are valid until we deploy https://github.com/gravitational/teleport/pull/5643.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,37 +1,113 @@
 # Changelog
 
-## 6.0.0-rc.1
+## 6.0.0
 
-**Note:** This version is a pre-release and is not recommended for production usage.
+Teleport 6.0 is a major release with new features, functionality, and bug fixes.
 
-This release of Teleport contains a number of improvements and bug fixes.
+The major features are [Database Access](https://goteleport.com/teleport/docs/preview/database-access/), open source role-based access control (RBAC), and exposing an easy to use API.
```
Comment (klizhentas) [inline CHANGELOG.md:7]: `We have implemented [Database Access](https://goteleport.com/teleport/docs/preview/database-access/), open sourced role-based access control (RBAC), and added official API and a Go client library.

File: CHANGELOG.md (hunk 4 above)
Comment (klizhentas) [inline CHANGELOG.md:28]: Remove `.mdx` extension, the link will not have it

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,37 +1,113 @@
...
+
+Database Access support was introduced in [RFD #11](https://github.com/gravitational/teleport/blob/master/rfd/0011-database-access.md).
+
+With Database Access users can connect to PostgreSQL and MySQL databases using
+short-lived certificates, configure SSO authentication and role-based access
+controls for databases, and capture SQL query activity in the audit log.
+
+#### Getting Started
+
+Configure Database Access from scratch following the [Getting Started](https://goteleport.com/teleport/docs/database-access/getting-started.mdx)
```
Comment (klizhentas) [inline CHANGELOG.md:23]: remove from scratch

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,37 +1,113 @@
...
-* Apply traits to kube, app and database labels: [#5597](https://github.com/gravitational/teleport/pull/5597)
-* Add `kube_public_addr` config field to `proxy_service`:[#5611](https://github.com/gravitational/teleport/pull/5611)
+Users can review the [6.0 milestone](https://github.com/gravitational/teleport/milestone/33?closed=1) on Github for more details.
 
-## 6.0.0-alpha.2
+## New Features
+
+### Database Access
+
+Database Access support was introduced in [RFD #11](https://github.com/gravitational/teleport/blob/master/rfd/0011-database-access.md).
```
Comment (klizhentas) [inline CHANGELOG.md:15]: Review the design of Database Access in [RFD #11](...)

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,37 +1,113 @@
...
+
+* [AWS RDS/Aurora PostgreSQL](https://goteleport.com/teleport/docs/database-access/guides/postgres-aws.mdx)
+* [AWS RDS/Aurora MySQL](https://goteleport.com/teleport/docs/database-access/guides/mysql-aws.mdx)
+* [Self-hosted PostgreSQL](https://goteleport.com/teleport/docs/database-access/guides/postgres-self-hosted.mdx)
+* [Self-hosted MySQL](https://goteleport.com/teleport/docs/database-access/guides/mysql-self-hosted.mdx)
+* [GUI clients](https://goteleport.com/teleport/docs/database-access/guides/gui-clients.mdx)
+
+#### Resources
+
+To learn more about configuring role-based access control for Database Access,
```
Comment (klizhentas) [inline CHANGELOG.md:36]: Same here and below, remove .mdx extensions from URLS

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,37 +1,113 @@
...
-* Rename Database Access service `database` role to `db`: [#5359](https://github.com/gravitational/teleport/pull/5359).
-* Fix `tsh login` failure when `--proxy` differs from actual proxy public address: [#5380](https://github.com/gravitational/teleport/pull/5380).
+RBAC support gives OSS administrators more granular access controls to servers and other resources with a cluster (like session recording access). An example of an RBAC policy could be: "admins can do anything, developers must never touch production servers and interns can only SSH into staging servers as guests"
 
-## 6.0.0-alpha.1
+In addition, some Access Workflow Plugins will now become available to open source users.
 
-**Note:** This version is a pre-release and is not recommended for production usage.
+* Access Workflows Golang SDK and API
+* Webhook
```
Comment (klizhentas) [inline CHANGELOG.md:56]: Remove webhook

**Review by klizhentas — APPROVED** [review]

**Comment by russjones** [comment]
@r0mant @webvictim Ready for re-review.

**Review by r0mant — APPROVED** [review]

**Comment by russjones** [comment]
@klizhentas You mentioned to wanted to add some migration notes to the CHANGELOG/release notes?

**Pull request merged**